catalog/meta-chatbot-inaccuracies.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Meta's AI bot trained on scientific info put out biased and inaccurate articles
```

Comment:
I feel like this could be more direct?

```suggestion
title: Meta's AI chatbot trained on scientific information produced racist and inaccurate articles
```